dash/frontend/src/app/modules/private/pages/falco/falco-settings/falco-settings.component.html

```diff
@@ -31,38 +31,45 @@ <h3>Notify about Anomalies</h3>
                   <div class ="row">
                     <h3>Notify no more than once every</h3>
                     &nbsp;&nbsp;
-                    <mat-form-field appearance="outline">
-                      <input matInput type="number" formControlName="dayInput">
+                    <mat-form-field appearance="outline" >
+                      <input [readonly]="isAnomalyDisabled" matInput type="number" formControlName="anomalyFrequency">
                     </mat-form-field>&nbsp;&nbsp;
                     <h3>day(s)</h3>
                   </div>
-
                   <div class ="row">
-                    <h3>Severity Levels to Include</h3>
+                    <h3>Severity Levels to Include</h3>&nbsp;&nbsp;
+                    <mat-form-field appearance="fill" [formGroup]="settingForm">
+                        <mat-select [disabled]="isAnomalyDisabled" formControlName="selectedPriorityLevels"
+                                    multiple>
+                          <mat-option *ngFor="let level of priorityLevels"
+                                      [value]="level">
+                            {{level}}
+                          </mat-option>
+                        </mat-select>
+                    </mat-form-field>
                   </div>
-                    <ul>
-                      <li *ngFor="let level of priorityLevels">
-                        <mat-checkbox [ngModel]="isChecked" matInput formControlName="selectedPriorityLevels" [value]="level">
-                        {{level}}
-                        </mat-checkbox>
-                      </li>
-                    </ul>
                 </div>
               </div>
 
               <div class="section-sublevel-1">
                 <div class="row">
                   <div>
-                    <mat-checkbox matInput formControlName="sendEmail"> </mat-checkbox>
+                    <mat-checkbox (change)="enableSummarySubLevelcheckbox($event)" matInput formControlName="sendNotificationSummary" disabled> </mat-checkbox>
```

Comment:
We don't want to show Falco settings for parts we haven't implemented yet. I know we discussed commenting this out but I don't see it commented out.

Reply:
The rules setting part were hidden as comment in the html file.